src/Stripe.net/Entities/StripeOutcome.cs

```diff
@@ -23,6 +23,12 @@ public class StripeOutcome : StripeEntityWithId
         [JsonProperty("risk_level")]
         public string RiskLevel { get; set; }
 
+        /// <summary>
+        /// Stripe’s evaluation of the riskiness of the payment. Possible values for evaluated payments are between 0 and 100.
+        /// </summary>
+        [JsonProperty("risk_score")]
+        public string RiskScore { get; set; }
```

Comment:
Type should be `int`, not `string`.

Reply:
@ob-stripe: Changed. sorry about this. Thanks for catching this.